manim/__main__.py

```diff
@@ -1,14 +1,178 @@
-from . import extract_scene
-from . import config
+import inspect
+import itertools as it
+import os
+import platform
+import subprocess as sp
+import sys
+import traceback
+import importlib.util
+
+from .config import file_writer_config
+from .scene.scene import Scene
+from .utils.sounds import play_error_sound
+from .utils.sounds import play_finish_sound
 from . import constants
+from .logger import logger
+
+
+def open_file_if_needed(file_writer):
```

Comment:
Should we leave the documentation of these functions to a proper documentation PR?